samples/ConsoleApp/Program.cs

```diff
@@ -13,6 +13,20 @@ private static async Task<int> Main()
         {
             using (var activity = MyActivitySource.StartActivity("Main"))
             {
+#if openTracingWrapper
```

Comment:
is there any reason why we do not want to have it always enabled?
the `openTracingWrapper` adds some additional complexity in scripts and project file as well

Reply:
yeah, we can have this on by default - I started defensively anticipating some problem since we don't have any there is no reason for the env var.